networker/networker.go

```diff
@@ -777,8 +777,8 @@ func (netw *Combined) stopAllowedIPv6Traffic() error {
 }
 
 func (netw *Combined) resetAllowlist() error {
-	// this is done in order to maintain the order of the firewall
-	// rules
+	// this is done in order to maintain the order of the firewall rules
+	log.Println(internal.DebugPrefix, "reset allow list")
```

Comment:
[minor] `Debug` is not appropriate for the release, maybe `InfoPrefix` would be better?

Reply:
yes, makes sense. I'll change to info.